tests/FSharp.Compiler.Service.Tests/FSharp.CompilerService.SurfaceArea.netstandard.expected

```diff
@@ -3604,8 +3604,8 @@ FSharp.Compiler.EditorServices.Structure+Collapse: System.String ToString()
 FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 ArrayOrList
 FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 Attribute
 FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 Comment
-FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 CompExpr
 FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 CompExprInternal
+FSharp.Compiler.EditorServices.Structure+Scope+Tags: Int32 ComputationExpr
```

Comment:
If we are renaming CompExpr --- ComputationExpr should we rename CompExprInternal ---> ComputationExprInternal?

Reply:
I removed CompExprInternal, it isn't used anywhere, thanks!